### PR TITLE
PORTALS-2144

### PR DESCRIPTION
**Pull request by yjyoo773**
Screen Shot 2022-03-30 at 10 32 33 AM
Screen Shot 2022-03-30 at 10 32 50 AM
Screen Shot 2022-03-30 at 10 35 24 AM
Screen Shot 2022-03-30 at 10 36 12 AM

### CODE REVIEW

File: src/components/AccountSettings.tsx [hunk 1]
```diff
@@ -61,17 +68,38 @@ const AccountSettings = (props: AccountSettingsProps) => {
                     mask,
                     accessToken
                 )
-
                 setUserProfile(bundle.userProfile)
                 setOrcid(bundle.ORCID)
                 setVerfied(bundle.isVerified)
+                setverificationSubmission(bundle.verificationSubmission)
+                setVerificationState(bundle.verificationSubmission?.stateHistory?.slice(-1)[0])
```
Comment (yjyoo773) [inline src/components/AccountSettings.tsx:75]: sets state for most recent Verification State submitted

File: src/components/AccountSettings.tsx [hunk 2]
```diff
@@ -61,17 +68,38 @@ const AccountSettings = (props: AccountSettingsProps) => {
...
         getData()
     }, [accessToken])
 
+    const ProfileValidationState = (verificationStateEnum: VerificationStateEnum | undefined) => {
+        const profileValidationStatus = () => {
+            if(verificationStateEnum === 'SUBMITTED'){
+                return <><img className="verifyBadgeIcon" src={CheckmarkBadgeDark} alt='CheckmarkBadgeDark'/>Pending Verification</>
+            } else if(verificationStateEnum === 'REJECTED' || verificationStateEnum ==='SUSPENDED'){
+                return (
+                    <div style={{border:'solid',padding:'24px', borderColor:'#4DB3B9'}}>
```
Comment (jay-hodgson) [inline src/components/AccountSettings.tsx]: Please replace inline styles with css class names (defined in the Account Settings scss)

File: src/components/AccountSettings.tsx [hunk 3]
```diff
@@ -27,6 +30,11 @@ const AccountSettings = (props: AccountSettingsProps) => {
     const [ isShowingWelcomeScreen, setIsShowingWelcomeScreen] = useState<boolean>(false)
     const [ updatedUsername, setUpdatedUsername] = useState<string>('')
     const [ updatedEmail, setUpdatedEmail] = useState<string>('')
+    const [ verificationSubmission, setverificationSubmission] = useState<VerificationSubmission>()
```
Comment (jay-hodgson) [inline src/components/AccountSettings.tsx]: should be "setVerificationSubmission"

File: src/components/AccountSettings.tsx [hunk 4]
```diff
@@ -222,6 +257,41 @@ const AccountSettings = (props: AccountSettingsProps) => {
...
+                className='verifiedModal bootstrap-4-backport'
+                show={showDialog}
+                onHide={()=>setShowDialog(false)}
+                animation={false}
+                backdrop='static'
+                centered
+            >   
+                <Modal.Body style={{margin:0, padding:'32px 0'}}>
+                    <Row>
+                        <Col sm={12} style={{textAlign:'center', marginBottom:'32px'}}><img src={VerifedAccount} alt="verified"/></Col>
```
Comment (jay-hodgson) [inline src/components/AccountSettings.tsx]: May as well be xs={12} since I think that's the default (and it's not overridden in larger displays).

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**